src/views/forms/FormNodeEdit/FormNodeEdit.vue

```diff
@@ -0,0 +1,91 @@
+                <Option value="add-node">Connect to a custom node</Option>
+              </Select>
+            </div>
+          </template>
+        </FormRow>
+        <FormRow>
+          <template v-slot:label> {{ $t('node_url') }}: </template>
+          <template v-slot:inputs>
+            <ValidationProvider
+              v-slot="{ errors }"
+              vid="nodeUrl"
+              :name="$t('node_url')"
+              :rules="validationRules.url"
+              tag="div"
+              class="inputs-container items-container"
+            >
+              <ErrorTooltip :errors="errors">
+                <AutoComplete
+                  v-model="formItems.nodeUrl"
+                  :data="customNodeData"
+                  class="auto-complete-size auto-complete-style"
+                  placeholder="input here"
+                  @on-search="handleInput"
+                  @on-select="getInfoFromUrl"
+                  @on-change="clearPrevious"
+                ></AutoComplete>
+              </ErrorTooltip>
+            </ValidationProvider>
+          </template>
+        </FormRow>
+        <FormRow>
+          <template v-slot:label> {{ $t('network_type') }}: </template>
+          <template v-slot:inputs>
+            <div class="inputs-container items-container">
+              <input
+                v-model="formItems.networkType"
+                class="input-size input-style"
+                :placeholder="$t('automatically_generated_by_node_url')"
+                disabled
+              />
+            </div>
+          </template>
+        </FormRow>
+        <FormRow>
+          <template v-slot:label> {{ $t('network_hash') }}: </template>
+          <template v-slot:inputs>
+            <div class="inputs-container items-container">
+              <input
+                v-model="formItems.networkHash"
+                class="input-size input-style"
+                :placeholder="$t('automatically_generated_by_node_url')"
+                disabled
+              />
+            </div>
+          </template>
+        </FormRow>
+      </div>
+      <div class="network-status">The current speed: <NetworkDelay :delay="nodeDelay" /></div>
+      <div class="text-prompt">
+        You can add node to the current network, the addition node that needs to satisfy the current network type,
+        otherwise you will receive an error message .
+      </div>
+      <div class="form-row button-container">
+        <button
+          class="button-style validation-button right-side-button"
+          type="submit"
+          :disabled="!formItems.networkHash"
+          @click="handleSubmit(onSubmit)"
+        >
+          {{ $t('confirm') }}
+        </button>
+      </div>
+    </ValidationObserver>
+  </div>
+</template>
+<script lang="ts">
+import { FormNodeEditTs } from './FormNodeEditTs'
+export default class FormNodeEdit extends FormNodeEditTs {}
```

Comment:
Since this form is about adding a custom node to the node list, I don't think the name `FormNodeEdit` is appropriate